packages/patternfly-4/react-core/src/components/Progress/Progress.docs.js

```diff
@@ -0,0 +1,34 @@
+import { Progress } from '@patternfly/react-core';
+import Simple from './examples/SimpleProgress';
+import LargeProgress from './examples/LargeProgress';
+import ProgressFailure from './examples/ProgressFailure';
```

Comment:
The variants (danger, success,etc) can probably be grouped similar to buttons. Additionally there is probably no need for a variant of every location since location does not differ from variant to variant. Same goes for sizes.

Reply:
It makes sense to have one object with all variants, but from future improvements I think it's much better to have them broken down as groups.

I was thinking about one object with groups, but that felt even more complicated.
```JS
ProgressVariant = {
  status: {
    danger: 'danger'
    info: 'info',
    success: 'siccess'
  },
  size: {
    lg: 'lg',
    md: 'md',
    xs: 'xs'
  },
  location: {
    top: 'top',
    inside: 'inside',
    outside: 'outside'
  }
}
```